Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -124,6 +130,18 @@ struct FloatingPanel<Content>: View where Content: View {
                     height = heightFor(detent: selectedDetent)
                 }
             }
+            .onReceive(NotificationCenter.default.publisher(for: UIResponder.keyboardDidShowNotification)) { notification in
+                withAnimation {
+                    keyboardHeight = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect)?.height ?? .zero
+                    height = heightFor(detent: .full) - keyboardHeight
+                }
+            }
+            .onReceive(NotificationCenter.default.publisher(for: UIResponder.keyboardDidHideNotification)) { _ in
+                withAnimation {
+                    keyboardHeight = .zero
+                    height = heightFor(detent: selectedDetent.wrappedValue) - keyboardHeight
```

Comment:
If we know the `keyboardHeight` is `.zero`, is it necessary to subtract it here?

Reply:
Good catch, no it shouldn't be.